package.json

```
@@ -42,8 +42,7 @@
     "prettier-check": "prettier --check \"**/**\"",
     "start": "react-scripts start",
     "build": "react-scripts --openssl-legacy-provider build",
```

What do you think about removing it from here too?

```suggestion
    "build": "react-scripts build",
```